share_app/src/main/java/org/odk/share/database/DatabaseContext.java

```diff
@@ -0,0 +1,37 @@
+package org.odk.share.database;
+
+import android.content.ContextWrapper;
+import android.database.DatabaseErrorHandler;
+import android.database.sqlite.SQLiteDatabase;
+
+import org.odk.share.application.Share;
+
+import java.io.File;
+
+/**
+ * Created by laksh on 6/13/2018.
+ */
+
+public class DatabaseContext extends ContextWrapper {
+    private String path;
+
+    public DatabaseContext(String path) {
+        super(Share.getInstance());
```

Comment:
Application context should be passed as a parameter instead of being accessed via singleton